frontend/src/features/mapSideBar/SidebarContents/AssociatedLandForm.tsx

```diff
@@ -137,6 +141,8 @@ export const valuesToApiFormat = (
   const apiValues = { ...values };
   apiValues.data.leaseExpiry = stringToNull(apiValues.data.leaseExpiry);
   apiValues.data.buildingTenancyUpdatedOn = stringToNull(apiValues.data.buildingTenancyUpdatedOn);
+  apiValues.data.evaluations = filterEmptyFinancials(apiValues.data.evaluations);
```

Comment:
I've removed the financials property which used to be used to map the assessed/netbook values to a single array. Now we just use the evaluations/fiscals directly as they are sent from the api, so we have to find the index of the one we want.